src/test/scala/uk/gov/hmrc/api/helpers/InitialiseReturnsSubmissionHelper.scala

```diff
@@ -17,11 +17,11 @@
 package uk.gov.hmrc.api.helpers
 
 import play.api.libs.ws.StandaloneWSResponse
-import uk.gov.hmrc.api.service.DisaReturnService
+import uk.gov.hmrc.api.service.InitialiseReturnsSubmissionService
+class InitialiseReturnsSubmissionHelper {
 
-class DisaSubmissionHelper {
-
-  val disaSubmissionService: DisaReturnService = new DisaReturnService
+  val disaInitialiseReturnsSubmissionService: InitialiseReturnsSubmissionService =
```

Comment:
Can we remove this helper if all it is doing is a straight call to the service method?